sql/core/src/test/scala/org/apache/spark/sql/connector/WriteDistributionAndOrderingSuite.scala

```diff
@@ -569,4 +785,26 @@ class WriteDistributionAndOrderingSuite
         fail("expected V2TableWriteExec")
     }
   }
+
+  private def orderedWritePartitioning(
+      writeOrdering: Seq[catalyst.expressions.SortOrder],
+      targetNumPartitions: Option[Int]): physical.Partitioning = {
+    targetNumPartitions match {
+      case Some(parts) => RangePartitioning(writeOrdering, parts)
+      case _ => RangePartitioning(writeOrdering, conf.numShufflePartitions)
+    }
+  }
+
+  private def clusteredTableDistribution(clustering: Array[Expression]): Distribution = {
+    Distributions.clustered(clustering)
```

Comment:
`clusteredTableDistribution` doesn't seem to be shorter than `Distributions.clustered` ...